modules/samples-operator-overview.adoc

```diff
@@ -47,3 +47,18 @@ the following upon deletion:
 
 Upon deletion of the samples resource, the Samples Operator recreates the
 resource using the default configuration.
+
+[id="samples-operator-bootstrapped"]
+== Samples Operator bootstrapped as removed
```

Comment:
the section heading should be something like "Sample Operator's use of management state"

Then we start with by default, we bootsrap as Managed, meaning the operator is actively managing its resources, the imagestreams and resource, and trying to keep the component active, i.e. install those samples and for imagestreams ensure the images are imported.

Then, dive into circumstances when we bootstrap as removed ...